tests/client/test_properties.py

```diff
@@ -1,36 +1,36 @@
-from httpx import URL, AsyncClient, Cookies, Headers, Timeout
+from httpx import URL, Client, Cookies, Headers, Timeout
 
 
 def test_client_base_url():
-    client = AsyncClient()
+    client = Client()
```

Comment:
I think these changes belong to a separate PR? Or most likely, this PR isn't up to date with `master`. :-)